File: bindings/java/src/main/java/io/ulalax/zlink/ZlinkVersion.java
/* SPDX-License-Identifier: MPL-2.0 */

package io.ulalax.zlink;

import io.ulalax.zlink.internal.Native;

public final class ZlinkVersion {
    private ZlinkVersion() {}

    public static int[] get() {
        return Native.version();
    }
}

File: bindings/dotnet/src/Zlink/ZlinkVersion.cs
// SPDX-License-Identifier: MPL-2.0

using Zlink.Native;

namespace Zlink;

public static class ZlinkVersion
{
    public static (int Major, int Minor, int Patch) Get()
    {
        NativeMethods.zlink_version(out int major, out int minor, out int patch);
        return (major, minor, patch);
    }
}
